MathProblems/2523_Closest_Prime_Numbers_in_Range.py

```
@@ -0,0 +1,92 @@
+import sys
+from sympy import *
+class Solution:
+    def closestPrimes(self, left: int, right: int) -> list[int]:
+        result=[] 
+        for i in range(left,right+1):
+            if isprime(i):
```

Ruff (0.11.9)

`isprime` may be undefined, or defined from star imports

(F405)